### PR TITLE
Button

### PR DESCRIPTION
**Pull request by ElishaSamPeterPrabhu**
Add fullwidth button Property
Modify alignment changes to meet fullwidth requirements #111

### CODE REVIEW

**Review by rthanga1 — CHANGES_REQUESTED** [review]
When `IsFullWidth` is disabled, the buttons still take full width. 

Screen Shot 2023-06-10 at 11 42 04 PM

**Review by rthanga1 — APPROVED** [review]

File: DemoApp/DemoApp/Views/TMButtonPage.xaml.cs [hunk 1]
```diff
@@ -47,4 +48,17 @@ private void ImagePositionChanged(object sender, CheckedChangedEventArgs e)
             _buttonPageViewModel.SelectedImageOption = radioButton.Value.ToString();
         }
     }
+
+    private void isFullWidth_Toggled(object sender, ToggledEventArgs e)
+    {
+        if (e.Value)
+        {
+            _buttonPageViewModel.IsFullWidthAlignment = LayoutOptions.FillAndExpand;
```
Comment (Sofiya-kumar) [inline DemoApp/DemoApp/Views/TMButtonPage.xaml.cs:56]: IsFullWidthAlignment is not a boolean property so please rename the valid property name

File: DemoApp/DemoApp/Views/TMButtonPage.xaml.cs [hunk 2]
```diff
@@ -11,10 +11,11 @@ public partial class TMButtonPage : ContentPage
 
     public TMButtonPage()
     {
-        _buttonPageViewModel = new ButtonPageViewModel();
+        _buttonPageViewModel = new ButtonPageViewModel();
 
         InitializeComponent();
-        BindingContext = _buttonPageViewModel;
+        BindingContext = _buttonPageViewModel;
+        _buttonPageViewModel.IsFullWidthAlignment = LayoutOptions.Start;
```
Comment (Sofiya-kumar) [inline DemoApp/DemoApp/Views/TMButtonPage.xaml.cs:18]: Move this initialize code to viewmodel

**Review by Sofiya-kumar — CHANGES_REQUESTED** [review]

**Review by rthanga1 — DISMISSED** [review]

File: DemoApp/DemoApp/Views/TMButtonPage.xaml.cs (hunk 2 above)
Comment (ElishaSamPeterPrabhu) [reply DemoApp/DemoApp/Views/TMButtonPage.xaml.cs]: Moved

File: DemoApp/DemoApp/Views/TMButtonPage.xaml.cs (hunk 1 above)
Comment (ElishaSamPeterPrabhu) [reply DemoApp/DemoApp/Views/TMButtonPage.xaml.cs]: Renamed

File: DemoApp/DemoApp/Views/TMButtonPage.xaml.cs [hunk 3]
```diff
@@ -47,4 +48,17 @@ private void ImagePositionChanged(object sender, CheckedChangedEventArgs e)
             _buttonPageViewModel.SelectedImageOption = radioButton.Value.ToString();
         }
     }
+
+    private void isFullWidth_Toggled(object sender, ToggledEventArgs e)
```
[the remark this answers is not in the document]
Comment (ElishaSamPeterPrabhu) [reply DemoApp/DemoApp/Views/TMButtonPage.xaml.cs]: Renamed

**Review by Sofiya-kumar — DISMISSED** [review]

**Review by rthanga1 — APPROVED** [review]

**Pull request merged**